src/components/Form/Form.md

```diff
@@ -1,3 +1,106 @@
+### Dynamic field issue
```

Comment:
When interacting with this form, my assumption would be that if I selected `I refuse, please let me submit`, it would allow me to submit the form, but that doesn't seem to be the case.